### PR TITLE
Translate "Getting Started" to Japanese

### PR DESCRIPTION
**Pull request by Pittan**
Hello! I translated "Getting started" page, and common `layout.html`.

### CODE REVIEW

**Comment by Pittan** [comment]
@inductor Hello! Would you review this PR?

File: src/i18n/ja/docs/getting_started.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# 🚀 Getting Started
+# 🚀 はじめに
 
-Parcel is a web application bundler, differentiated by its developer experience. It offers blazing fast performance utilizing multicore processing, and requires zero configuration.
+Parcel は、開発者の経験によって生まれた Web アプリケーションバンドラです。設定不要で、マルチコア処理を利用した驚異的な高速パフォーマンスを提供します。
 
-First install Parcel using Yarn or npm:
+まずは Yarn または npm で Parcel をインストールしましょう。
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
まずは yarn または npm で Parcel をインストールしましょう。
```

In Japanese, in my opinion, `yarn` should be better than `Yarn` (all lower cases)

File: src/i18n/ja/docs/getting_started.md [hunk 2]
```diff
@@ -16,21 +16,21 @@ npm:
 npm install -g parcel-bundler
 ```
 
-Create a package.json file in your project directory using:
+あなたのプロジェクトディレクトリに以下のコマンドを使って package.json を作成しましょう。
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
プロジェクトルートに以下のコマンドを使って package.json を作成しましょう。
```

あなたの can be omitted. プロジェクトルート sounds more familiar to me but it is an optional fix :)

File: src/i18n/ja/docs/getting_started.md [hunk 3]
```diff
@@ -16,21 +16,21 @@ npm:
...
 
-or
+または
 
 ```bash
 npm init -y
 ```
 
-Parcel can take any type of file as an entry point, but an HTML or JavaScript file is a good place to start. If you link your main JavaScript file in the HTML using a relative path, Parcel will also process it for you, and replace the reference with a URL to the output file.
+Parcel はどんな種類のファイルでもエントリーポイントとして扱うことができますが、HTML または JavaScript ファイルから始めるのがぴったりです。HTML ファイルから相対パスでメインとなる JavaScript ファイルをリンクすると、Parcel はそれを処理し、参照を出力したファイルの URL に置き換えます。
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
Parcel はどんな種類のファイルでもエントリーポイントとして扱うことができますが、HTML または JavaScript ファイルから始めるのがよいでしょう。HTML ファイルから相対パスでメインとなる JavaScript ファイルをリンクすると、Parcel はそれを処理し、参照を出力したファイルの URL に置き換えます。
```

File: src/i18n/ja/docs/getting_started.md [hunk 4]
```diff
@@ -44,37 +44,37 @@ Next, create an index.html and index.js file.
...
 
 ```bash
 parcel index.html
 ```
 
-Now open http://localhost:1234/ in your browser. If hot module replacement isn't working you may need to [configure your editor](hmr.html#safe-write). You can also override the default port with the `-p <port number>` option.
+では、ブラウザで http://localhost:1234/ を開いてみましょう。もし、ホットモジュールリプレイスメントが動いていない場合は[エディタの設定を編集](hmr.html#safe-write)する必要があるかもしれません。 `-p <ポート番号>` オプションを使うことでデフォルトのポート番号を上書きすることができます。
 
-Use the development server when you don't have your own server, or your app is entirely client rendered. If you do have your own server, you can run Parcel in `watch` mode instead. This still automatically rebuilds as files change and supports hot module replacement, but doesn't start a web server.
+自分でサーバーを持っていない場合、あるいはあなたのアプリケーションがすべてクライアントでレンダリングされる場合には開発用サーバを使いましょう。自分でサーバーを持っている場合には、代わりに Parcel を `watch` モードで実行することができます。ファイルが変更されると自動で再ビルドを行い、ホットモジュールリプレースメントをサポートしますが、Web サーバーは起動しません。
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
自分でサーバーを持っていない場合、あるいはアプリケーションがすべてクライアントでレンダリングされる場合には、Parcel の開発用サーバを使いましょう。自分でサーバーを持っている場合には、代わりに Parcel を `watch` モードで実行することができます。ファイルが変更されると自動で再ビルドを行い、ホットモジュールリプレースメントをサポートしますが、Web サーバーは起動しません。
```

In English `the development server` means the development server that Parcel contains

File: src/i18n/ja/docs/getting_started.md [hunk 5]
```diff
@@ -83,29 +83,29 @@ _NOTE:_ In case you have a file structure like this:
...
+本番用のビルドを行うときは、 `build` モードがファイルの監視を無効にして、一度だけビルドを行います。 詳細は[本番](production.html)を参照してください。
 
-### Adding parcel to your project
+### Parcel をプロジェクトに追加する
 
-Sometimes it's not possible to install Parcel globally e.g. if you're building on someone else's build agent or you want to use a CI to build your project programmatically. In this case, you can install and run Parcel as a local package.
+時には Parcel をグローバルインストールできないこともあるでしょう。（例：他人のマシンや CI を使ってプロジェクトを自動でビルドしたいとき）こういう場合には、ローカルパッケージとして Parcel をインストールすることができます。
 
-To install with Yarn:
+Yarn でインストールする方法
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
yarn でインストールする方法
```

File: src/i18n/ja/docs/getting_started.md [hunk 6]
```diff
@@ -83,29 +83,29 @@ _NOTE:_ In case you have a file structure like this:
...
 
-To install with NPM:
+npm でインストールする方法
 
 ```bash
 npm install parcel-bundler --save-dev
 ```
 
-Then, add these tasks scripts to your project, by modifying your `package.json`:
+それから、あなたのプロジェクトの `package.json` にこれらのタスクスクリプトを追加します。
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
それから、プロジェクトの `package.json` にこれらのタスクスクリプトを追加します。
```

File: src/i18n/ja/docs/getting_started.md [hunk 7]
```diff
@@ -116,16 +116,16 @@ Then, add these tasks scripts to your project, by modifying your `package.json`:
 }
 ```
 
-Then, you will be able to run it:
+そうすると、以下のように実行することができます
```
Comment (inductor) [inline src/i18n/ja/docs/getting_started.md]:
```suggestion
すると、以下のように実行することができるようになります
```

File: src/i18n/ja/docs/getting_started.md (hunk 2 above)
Comment (Pittan) [reply src/i18n/ja/docs/getting_started.md]: starting from プロジェクトルート sounds good!

**Pull request merged**